scipy/spatial/transform/_rotation.pyx

```diff
@@ -2836,3 +2831,7 @@ class Slerp:
             result = result[0]
 
         return result
+
+    @property
+    def rotvecs(self):
+        return (self.rotations.inv() * rotations[1:]).as_rotvec()
```

Comment:
This is a change of behaviour as it would recompute at every call. Can you give the rational of this change?

Reply:
It makes more sense to me philosophically to be a property rather than an attribute calculated during `__init__`. Similarly to how `as_matrix()`, `as_euler()`, etc. are `Rotation` methods, not `.matrix`, `.euler`, etc. attributes computed once during initialization. I will modify my statement about not changing behavior though, good point.

Reply:
I am afraid we cannot change part of a public API's behaviour if there are no imperative reasons. Our policy is to be conservative and err on the side of caution about API changes and possible backward incompatibilities. Someone could be relying on this exact behaviour for some reason, have a call to this attribute in a hot loop and the change would introduce unnecessary/unintended slow down.

But if there is another argument to make the change, then sure we can consider it 😃